src/lib/jdk/KeyTool.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as util from '../util';
```

Comment:
This is more of a style thing, and I'm happy to be outvoted, but I'd prefer it if we explicitly listed our imports instead of using `import *`.

Reply:
SG - will change to import each individual one.

Reply:
Sounds like a good one. done.